_dev/apps/ui/src/lib/i18n.ts

```diff
@@ -53,7 +53,12 @@ export const loadLanguageAsync = async (lang: string): Promise<string> => {
     return Promise.resolve(setI18nLanguage(lang));
   }
 
-  // If the language hasn't been loaded yet
+  // If the language is not supported, fallback in English
+  if (!Object.keys(translationFiles).includes(lang)) {
```

Comment:
there's a simpler way to do this:
```typescript
if( !translationFiles[lang] ) {
```
or, as it seems to be a function, you can be more specific with a 
```typescript
if( typeof translationFiles[lang] !=="function" ) {
```

Reply:
Indeed. The change has just been pushed